src/libraries/Common/tests/CoreFx.Private.TestUtilities/System/PlatformDetection.Unix.cs

```diff
@@ -328,8 +329,20 @@ private static class libc
             [DllImport("libc", CharSet = CharSet.Ansi, CallingConvention = CallingConvention.Cdecl)]
             public static extern IntPtr gnu_get_libc_version();
 
-            [DllImport("System.Globalization.Native", SetLastError = true)]
-            public static extern int GlobalizationNative_GetICUVersion();
+            public unsafe static int GetICUVersion()
```

Comment:
This should not be inside `class libc`. It has nothing to do with libc. It can be in the place that calls `libc.GetICUVersion` above.

Reply:
👍